checkov/terraform/checks/graph_checks/aws/ConfigRecorderRecordsAllGlobalResources.yaml

```
@@ -0,0 +1,29 @@
+metadata:
+  id: "CKV2_AWS_48"
+  name: "Ensure AWS Config must record all possible resources"
+  category: "LOGGING"
+definition:
+  and:
+    - cond_type: filter
+      attribute: resource_type
+      operator: within
+      value:
+        - aws_config_configuration_recorder_status
+    - cond_type: connection
+      resource_types:
+        - aws_config_configuration_recorder_status
+      connected_resource_types:
+        - aws_config_configuration_recorder
+      operator: exists
+    - cond_type: attribute
+      resource_types:
+        - aws_config_configuration_recorder
+      attribute: recording_group.include_global_resource_types
+      operator: equals
+      value: "true"
+    - cond_type: attribute
+      resource_types:
+        - aws_config_configuration_recorder_status
+      attribute: is_enabled
+      operator: equals
+      value: "true"
```

```suggestion
  cond_type: attribute
  resource_types:
    - aws_config_configuration_recorder
  attribute: recording_group.include_global_resource_types
  operator: equals
  value: "true"
```
this should be enough, due to `CKV2_AWS_45` already checking it enabled